tests/test_random.py

```diff
@@ -120,6 +120,18 @@ def dfunction(x):
 dLookupTableResult = (0.23721845680847731, 0.42913599265739233, 0.86176396813243539)
 # File with the same values
 dLookupTableFile = os.path.join('random_data','dLookupTable.dat')
+# x and p arrays with and without a small (epsilson) step
+dx_eps = np.arange(6)
+dp1_eps = np.zeros(dx_eps.shape)
+dp2_eps = np.zeros(dx_eps.shape)
+eps = np.finfo(dp1_eps[0].dtype).eps
+dp1_eps[0] = 0.5
+dp2_eps[0] = 0.5
+dp1_eps[-1] = 0.5
+dp2_eps[-2] = eps
+dp2_eps[-1] = 0.5-eps
+dLookupTableEps1 = galsim.LookupTable(x=dx_eps, f=dp1_eps, interpolant='linear')
+dLookupTableEps2 = galsim.LookupTable(x=dx_eps, f=dp2_eps, interpolant='linear')
```

Comment:
These aren't getting used, so is the problem that it would die when trying to initialize the LookupTable?